SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -39,7 +39,7 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject
 	void SendLogMessage(const uint8_t Level, const char* LoggerName, const char* Message);
 	void SendComponentInterest(Worker_EntityId EntityId, const TArray<Worker_InterestOverride>& ComponentInterest);
 	FString GetWorkerId() const;
-	TArray<FString> GetWorkerAttributes() const;
+	const TArray<FString>& GetWorkerAttributes() const;
```

Comment:
Drive-by changes to the code from https://github.com/spatialos/UnrealGDK/pull/570. I could move this to a separate PR if anyone feels strongly about it.